macro/macroutils.py

```diff
@@ -92,14 +92,20 @@ def sub_link(macro, page, sub, title=None):
         title = sub
     return Page(macro.request, '%s/%s'%(page, sub)).link_to(macro.request, text=title)
 
-def wiki_url(macro, page,shorten=None,querystr=None):
+def wiki_url(macro, page,shorten=None,querystr=None, relative=False, raw=False):
     """
     Create link to ROS wiki page
+    @param raw: Return a raw html link instead of a wiki link. (This will avoid moin moin link checking.)
     """
     if not shorten or len(page) < shorten:
         page_text = page
     else:
         page_text = page[:shorten]+'...'
+    if raw:
+        ret = '<a href="'
+        ret += Page(macro.request, page).url(macro.request, querystr=querystr, relative=relative)
```

Comment:
Did you consider the use of `link_to_raw` here? (perhaps it's not available for our version)

Reply:
It definitely caught my attention, but my impression was that this still was a wiki link not the url for html. The only thing that `link_to` does after calling `link_to_raw` is append the attachment link.

I think the functionality we're trying to avoid is inside the `link_tag` method which is still inside the `link_to_raw` invocation when it starts invoking the formatters etc. It's quite possible that with the right kw arguments the formatter behavior could be adjusted to use that method.

Reply:
> The only thing that link_to does after calling link_to_raw is append the attachment link.

Before calling to `link_to_raw`, `link_to` will [check if the page exists](https://bitbucket.org/thomaswaldmann/moin-1.9/src/a6283e1898696298309f5b59d328307480b1dbda/MoinMoin/Page.py?at=default&fileviewer=file-view-default#Page.py-797) and that looks to me like where the overhead comes in.

Either way, since we're looking to save time on the response it can't hurt to bypass as much processing as possible so I'm in favour of this change; if we need more e.g. escaping one day we can revisit.